cli/src/semgrep/commands/scan.py

```diff
@@ -547,6 +548,16 @@ def convert(
         hidden=True,
         help="Enable support for secret validation. Requires Semgrep Secrets, contact support@semgrep.com for more informationon this.",
     ),
+    optgroup.option(
+        "--diff-depth",
+        type=int,
+        default=DEFAULT_DIFF_DEPTH,
+        help="""
+            The depth of the differential scan, the number of steps (both in the
+            caller and callee sides) from the targets in the call graph tracked
+            by the deep preprocessor. Only applied in differential scan mode.
```

Comment:
I would make sure to mention pro and/or interfile here

Reply:
done